patches/CloudCompare-PythonRuntime.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/plugins/private/CloudCompare-PythonRuntime/src/PythonInterpreter.cpp b/plugins/private/CloudCompare-PythonRuntime/src/PythonInterpreter.cpp
+index 85a5a8e..6683a30 100644
+--- a/plugins/private/CloudCompare-PythonRuntime/src/PythonInterpreter.cpp
++++ b/plugins/private/CloudCompare-PythonRuntime/src/PythonInterpreter.cpp
+@@ -198,13 +198,13 @@ void PythonInterpreter::initialize(const PythonConfig &config)
+         }
+     };
+ 
+-    char soName[25];
+-    snprintf(soName, 24, "libpython%d.%d.so", PythonVersion.major, PythonVersion.minor);
++    char soName[60];
++    snprintf(soName, 59, "/usr/lib/x86_64-linux-gnu/libpython%d.%d.so.1.0", PythonVersion.major, PythonVersion.minor);
```

Comment:
Is it not possible to make this work without hard-coding the path? It would be nice if we eventually can compile this on arm64.

Reply:
Yes, it's possible but I plan this for further release. I should add some modifications to CCPython-Runtime.